pkg/dash/config.go

```diff
@@ -29,8 +32,6 @@ func ParseEnvironment() (*Config, error) {
 			u.User = url.UserPassword(user, token)
 			config.GrafanaURL = u.String()
 		}
-	} else {
-		return nil, errors.New("Must set GRAFANA_URL environment variable")
```

Comment:
Good call here. It'll be nice to skip setting `GRAFANA_URL` for running local-only operations.